data/hai/hai ships.txt

```diff
@@ -986,7 +1001,7 @@ ship "Sea Scorpion" "Sea Scorpion (Shredder)"
 		"Hai Williwaw Cooling" 4
 		"Hyperdrive"
 		"Pulse Cannon" 2
-		"Pulse Rifle" 15
+		"Pulse Rifle" 36
```

Comment:
Is this necessary? You can't complete the mission if you capture this ship.

Reply:
you can capture every single one of these ships and complete the mission, all that is required is to disable, not to save
this is not a standard duel, as the UHai warn in the convo
also yes it is very much necessary to show they care about the prototypes

Reply:
`npc disable` means that you have to leave the npcs disabled to complete the mission.

Reply:
npc disable means you must have disabled them at one point, and even if they get repaired afterwards it's still valid

Reply:
OK, thanks.